manifests/m/MichaelJolley/RandomDadJokesForCmdPal/0.1.0.0/MichaelJolley.RandomDadJokesForCmdPal.installer.yaml

```
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: MichaelJolley.RandomDadJokesForCmdPal
+PackageVersion: 0.1.0.0
+Platform:
+- Windows.Universal
+- Windows.Desktop
+MinimumOSVersion: 10.0.19041.0
+InstallerType: msix
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.WindowsAppRuntime.1.7

```

```suggestion
  - PackageIdentifier: Microsoft.WindowsAppRuntime.1.7
  - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.9
```
Apologies for the multiple messages - I didn't see the pre-existing `Dependencies` property when originally suggesting the change.